website/content/docs/enterprise/license.mdx

```diff
@@ -0,0 +1,191 @@
+    2021-03-29T15:09:30.164-0400 [INFO]  client.plugin: plugin manager finished: plugin-type=driver
+    2021-03-29T15:09:30.164-0400 [INFO]  client.plugin: shutting down plugin manager: plugin-type=csi
+    2021-03-29T15:09:30.164-0400 [INFO]  client.plugin: plugin manager finished: plugin-type=csi
+    2021-03-29T15:09:30.164-0400 [DEBUG] client.server_mgr: shutting down
+    2021-03-29T15:09:30.164-0400 [INFO]  nomad: shutting down server
+    2021-03-29T15:09:30.164-0400 [WARN]  nomad: serf: Shutdown without a Leave
+    2021-03-29T15:09:30.165-0400 [DEBUG] nomad: shutting down leader loop
+    2021-03-29T15:09:30.165-0400 [INFO]  nomad: cluster leadership lost
+    2021-03-29T15:09:30.170-0400 [INFO]  agent: shutdown complete
+```
+
+Since this was a temporary license, when
+the temporary license expires, the agent shuts down.
+
+### Valid, Non-Temporary Licenses
+
+License expiry is handled differently for valid enterprise licenses. Nomad
+licensing will continue to log about the expiring license above, but when the
+license fully expires (the Termination Time is reached) the server _will not_
+shut down. Instead, of shutting down, enterprise functionality will become limited. Only
+read operations on enterprise endpoints will be supported, and write operations
+will return an error.
+
+~> **Note:** When an enterprise server starts and the license is expired, Nomad
+will wait for a short grace period to apply a valid license before shutting
+down.
+
+## Setting the License
+
+See the server [license configuration] reference documentation on all the
+options to set an enterprise license.
+
+When setting a Nomad Enterprise license there are two options to pick from. You
+can set the license via the CLI or API after the server is running, or Nomad
+can automatically load the file from disk or environment when it starts.
+
+To set the license via CLI, see the [license command] documentation. To set the
+license programmatically see the [license endpoint] API documentation.
+
+To configure Nomad to load the license from disk or environment see the server
+[license configuration].
+
+## Operating Nomad Enterprise with a License
+
+In order to immediately alert operators of a bad configuration setting, if a
+license configuration option is a completely invalid license, the nomad server
+will exit with an error.
+
+```
+NOMAD_LICENSE=misconfigured nomad agent -dev
+==> No configuration files loaded
+==> Starting Nomad agent...
+==> Error starting agent: server setup failed: failed to initialize enterprise licensing: a file license was configured but the license is invalid: error decoding version: expected integer
+```
+
+Some Nomad servers are controlled with a level of automation or could be part
+of an autoscaling group. If an operator accidentally has an old, expired
+license set as the disk or environment license, the server will emit a warning
+log, but not exit in case a valid license exists in raft. If a valid license
```

Comment:
This makes it sounds like "just in case", so I'd change this to either "in the case" or more simply:
```suggestion
log, but not exit if a valid license exists in raft. If a valid license
```